deploy/helm/templates/addons/prometheus-addon.yaml

```diff
@@ -17,14 +17,15 @@ spec:
     {{- if hasPrefix "oci://" .Values.addonChartLocationBase }}
     chartLocationURL: {{ .Values.addonChartLocationBase }}/prometheus
     {{- else }}
-    chartLocationURL: {{ .Values.addonChartLocationBase }}/prometheus-15.16.1.tgz
+    chartLocationURL: {{ .Values.addonChartLocationBase }}/prometheus-{{ default .Chart.Version .Values.versionOverride }}.tgz
     {{- end }}
 
     installOptions:
       {{- if hasPrefix "oci://" .Values.addonChartLocationBase }}
       version: 15.16.1
```

Comment:
fixed value is not right, ref to other addon.

Reply:
done